hms-app/src/pages/IdeaPortal.tsx

```diff
@@ -202,35 +76,33 @@ function IdeaPortal() {
         />
       </Group>
 
-      {isHackathonError && (
-        <div>
-          <h3>Error loading hackathons</h3>
-          <p>something went wrong.</p>
-        </div>
-      )}
-      {isHackathonLoading && (
-        <div>
-          <h3>Hackathon details are loading...</h3>
-        </div>
-      )}
+      <GetRelevantIdeas
```

Comment:
naming this thing `GetRelevantIdeas` seems a bit odd.  A component should be named a bit more like a noun and not a verb.  So maybe `RelevantIdeasList`